Dockerfile

```diff
@@ -4,18 +4,164 @@ MAINTAINER Lisa Seelye <lisa@thedoh.com>
+  cp Modules/Setup.dist Modules/Setup && \
+  MODULES="_bisect _collections _csv _datetime _elementtree _functools _heapq _io _md5 _posixsubprocess _random _sha256 _sha512 _socket _struct _weakref array binascii cmath datetime fcntl grp itertools math mmap operator parser readline resource select spwd syslog termios time unicodedata zlib" && \
+  for mod in $MODULES; do echo "enabling ${mod}" ; sed -i -e "s/^#${mod}/${mod}/" Modules/Setup;  done && \
+  echo '_json _json.c' >> Modules/Setup && \
+  echo '_multiprocessing _multiprocessing/multiprocessing.c _multiprocessing/semaphore.c _multiprocessing/socket_connection.c' >> Modules/Setup && \
+  sed -i '1i *static*' Modules/Setup && \
+  sed -i \
+      -e "s|^zlib zlibmodule.c|zlib zlibmodule.c -I/build/zlib-${ZLIB_VERSION} -L/build/zlib-${ZLIB_VERSION} -lz|" \
+      -e "s|^readline readline.c|readline readline.c -I/build/readline-${READLINE_VERSION} -L/build/readline-${READLINE_VERSION} -L/build/termcap-${TERMCAP_VERSION} -lreadline -ltermcap|" \
+      Modules/Setup && \
+  echo "" >> Modules/Setup && \
+  echo "_lsprof rotatingtree.c _lsprof.c" >>Modules/Setup && \
+  echo "pyexpat expat/xmlparse.c expat/xmlrole.c expat/xmltok.c pyexpat.c -I\$(srcdir)/Modules/expat -DHAVE_EXPAT_CONFIG_H -DUSE_PYEXPAT_CAPI -DHAVE_SYSCALL_GETRANDOM" >>Modules/Setup && \
+  echo "_psutil_linux psutil/_psutil_common.c psutil/_psutil_posix.c psutil/_psutil_linux.c -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=__PSVER__ -DPSUTIL_LINUX=1" >>Modules/Setup && \
+  echo "_psutil_posix psutil/_psutil_common.c psutil/_psutil_posix.c                        -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=__PSVER__ -DPSUTIL_LINUX=1" >>Modules/Setup && \
+  echo "crypt cryptmodule.c  -lcrypt" >>Modules/Setup && \
+  patch --ignore-whitespace -p1 < /build/cpython-enable-openssl.patch && \
+  echo "Patching for https://bugs.python.org/issue7938" && \
+  patch --ignore-whitespace -p0 < /build/BPO-7938_pr4338-fix-makesetup-script.patch && \
+  sed -i \
+      -e "s|^SSL=/build/openssl-TKTK|SSL=/build/openssl-${OPENSSL_VERSION}|" \
+      -e "s|__PSVER__|${psversion}|g"                                        \
+      Modules/Setup && \
+  cat Modules/Setup && \
+  CC='/opt/cross/x86_64-linux-musl/bin/x86_64-linux-musl-gcc -static -fPIC' \
+    CXX='/opt/cross/x86_64-linux-musl/bin/x86_64-linux-musl-g++ -static -static-libstdc++ -fPIC' \
+    LD=/opt/cross/x86_64-linux-musl/bin/x86_64-linux-musl-ld \
+    ./configure \
+      --disable-shared \
+      --with-openssl=/build/openssl-${OPENSSL_VERSION} && \
+  (make --trace -j $(nproc --all) LDFLAGS="-static" LINKFORSHARED=" " || true) && \
+  /opt/cross/x86_64-linux-musl/bin/x86_64-linux-musl-strip python && \
+  cp $(find . -name _sysconfigdata.py -print) Lib && \
+  cp -r /build/psutil/python_files/psutil Lib && \
+  cd Lib && zip -r ../python2.7.zip .
 
 ###
 FROM scratch
 MAINTAINER Lisa Seelye <lisa@thedoh.com>
 WORKDIR /
-COPY --from=0 /output/python* /
 
+ARG PYTHON_BASE_VERSION=2.7.15
```

Comment:
It is unfortunate that the `PYTHON_BASE_VERSION` and `PYTHON_RC` build args need to be duplicated in this `scratch` base but it makes sense to do so.